features/signals_railway_signals.yaml

```diff
@@ -3815,232 +3783,253 @@ features:
       - { tag: 'railway:signal:train_protection:form', value: 'sign' }
+      - { tag: 'railway:signal:train_protection:type', value: 'block_marker' }
 
-  - description: ETCS 10
+  - description: Wskaźnik lokalizacji ETCS (WETCS11)
     country: PL
     icon:
-      match: 'railway:signal:train_protection'
+      match: 'railway:signal:position'
       cases:
-        - { regex: '^left$', value: 'pl/wetcs10-left' }
-      default: 'pl/wetcs10-right'
+        - { regex: '^left$', value: 'pl/wetcs11-right' }
+        - { regex: '^overhead$', value: 'pl/wetcs11-down' }
+      default: 'pl/wetcs11-left'
     tags:
-      - { tag: 'railway:signal:train_protection', value: 'PL-PKP:wetcs10' }
+      - { tag: 'railway:signal:train_protection', value: 'PL-PKP:wetcs11' }
       - { tag: 'railway:signal:train_protection:form', value: 'sign' }
-      - { tag: 'railway:signal:train_protection:type', value: 'block_marker' }
+      - { tag: 'railway:signal:train_protection:type', value: 'location_marker' }
 
-  - description: WKD W2
+  - description: Wskaźnik WKD W2
     country: PL
     icon: { default: 'pl/wkd/w2' }
     tags:
       - { tag: 'railway:signal:distant', value: 'PL-WKD:w2' }
       - { tag: 'railway:signal:distant:form', value: 'sign' }
 
-  - description: miejsce zatrzymania czoła pociągu Wm 4
+  - description: Miejsce zatrzymania czoła pociągu (Wm4)
     country: PL
-    icon: { default: 'pl/ztm-mw/wm4' }
+    icon: { default: 'pl/metro/wm4' }
     tags:
-      - { tag: 'railway:signal:stop', value: 'ZTM-MW:wm4' }
+      - { tag: 'railway:signal:stop', value: 'PL-metro:wm4' }
       - { tag: 'railway:signal:stop:form', value: 'sign' }
 
-  - description: rozpocząć hamowanie przed peronem Wm 16
+  - description: Rozpocząć hamowanie przed peronem (Wm16)
     country: PL
-    icon: { default: 'pl/ztm-mw/wm16' }
+    icon: { default: 'pl/metro/wm16' }
     tags:
-      - { tag: 'railway:signal:station_distant', value: 'ZTM-MW:wm16' }
+      - { tag: 'railway:signal:station_distant', value: 'PL-metro:wm16' }
       - { tag: 'railway:signal:station_distant:form', value: 'sign' }
 
-  - description: Wm 17
+  - description: Wskaźnik ukresu (Wm17)
     country: PL
-    icon: { default: 'pl/ztm-mw/wm17' }
+    icon: { default: 'pl/metro/wm17' }
     tags:
-      - { tag: 'railway:signal:fouling_point', value: 'ZTM-MW:wm17' }
+      - { tag: 'railway:signal:fouling_point', value: 'PL-metro:wm17' }
 
-  - description: D6 Tarcza zwolnić bieg
+  - description: Tarcza zwolnić bieg (D6)
     country: PL
     icon:
       match: 'railway:signal:speed_limit_distant:speed'
       # TODO update signs from https://wiki.openstreetmap.org/wiki/Pl:OpenRailwayMap/Tagging_in_Poland#Tarcza_zwolnić_bieg_D_6
       # TODO handle dwarf signals
       cases:
-        - { regex: '^[1-9]0$', value: 'pl/d6-{}', example: 'pl/d6-{70}' }
+        - { regex: '^([1-9]|1[0-9]|2[0-4])0$', value: 'pl/d6-{}', example: 'pl/d6-{70}' }
```

Comment:
For this one there are only icons for speeds 10-90. So this will fail to make icons for higher speeds.